builtins/builtins.go

```diff
@@ -34,6 +35,15 @@ func Int2Char(args ...runtime.Object) runtime.Object {
 	return runtime.Errorf("Argument is out of range. Range is from 0 to 127. Int = %s", n.String())
 }
 
+func Str2Int(args ...runtime.Object) runtime.Object {
+	rstring := args[0].(*runtime.String)
+	i, err := strconv.Atoi(string(rstring.Value))
```

Comment:
Converting `rstring.Value` to a string makes assumptions about the Value type. We can prevent this kind of "data type coupling" by using the dedicated `rstring.String` method:

```suggestion
	i, err := strconv.Atoi(rstring.String()))
```